docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        for i in range(split_idx, k):
</pre><p>Instead of this <code>for</code> loop, can we simplify and do:</p><pre class="ql-syntax" spellcheck="false">PL = core._idx_to_mp(IL, T, m, check_neg=False) 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
That's right! Will revise it 👍